docs/getting-started-guides/docker.md

```diff
@@ -113,7 +113,13 @@ Now run `docker ps` you should see nginx running.  You may need to wait a few mi
 ### Expose it as a service
 
 ```shell
-kubectl expose rc nginx --port=80
+kubectl expose deployment nginx --port=80
+```
+
+If running with docker-machine, you want to expose the deployment on the docker-machine ip as well, so use
+
+```shell
+kubectl expose deployment nginx --port=80 --external-ip=$(docker-machine ip)
```

Comment:
This is great.  Can you update this section:

On OS X, since docker is running inside a VM, run the following command instead: shell docker-machine ssh `docker-machine active` curl $ip

To say something like
If running with docker-machine, you will need to use the EXTERNAL_IP found with `kubectl get svc nginx --template={{.spec.externalIPs}}`